config/locales/en.yml

```diff
@@ -292,21 +292,25 @@ en:
         link_text: Check what you need to do
         title: Brexit
       ukraine:
-        title: Invasion of Ukraine
         links:
-        - label: "UK visa support for Ukrainian nationals"
-          href:  "https://www.gov.uk/guidance/support-for-family-members-of-british-nationals-in-ukraine-and-ukrainian-nationals-in-ukraine-and-the-uk"
-        - label: "Move to the UK if you're coming from Ukraine"
-          href:  "https://www.gov.uk/guidance/move-to-the-uk-if-youre-from-ukraine"
-        - label: "Homes for Ukraine: record your interest"
-          href:  "https://www.gov.uk/register-interest-homes-ukraine"
-        - label: "Find out about the UK’s response"
-          href:  "https://ukstandswithukraine.campaign.gov.uk/"
+        - label: UK visa support for Ukrainian nationals
+          href: https://www.gov.uk/guidance/support-for-family-members-of-british-nationals-in-ukraine-and-ukrainian-nationals-in-ukraine-and-the-uk
+        - label: Move to the UK if you're coming from Ukraine
+          href: https://www.gov.uk/guidance/move-to-the-uk-if-youre-from-ukraine
+        - label: 'Homes for Ukraine: record your interest'
+          href: https://www.gov.uk/register-interest-homes-ukraine
+        - label: Find out about the UK’s response
+          href: https://ukstandswithukraine.campaign.gov.uk/
+        title: Invasion of Ukraine
       world_locations: World locations
     search_box:
       input_title: Search
       label: Search on GOV.UK
       search_button: Search
+    share_links:
+      all_opens_in_new_tab: Sharing will open the page in a new tab
+      open:
```

Comment:
All looks good except for this "open" key in the translations which seems to be redundant. If we can remove that we're good to go.